### PR TITLE
#333- Updated Landing Page V2

### LINKED ISSUE #333: Landing Page V2
- [ ] Update main header message to 'Open-source community for creators' (all **bold**)
- [ ] Update main subtext to 'Learn to code, collaborate on projects, gain experience, build a community, and earn coins by contributing.'
- [ ] Remove images of the app & replace it with the illustration attached to the ticket
- [ ] Change background color of the 3-step block to Sail **Blue** 900 (#131F41)
- [ ] Link colors in 3-step block is Sail **Blue** 300 (#7DABF8)
- [ ] On the 3rd step, change the wording 'Earn Points' to 'Earn Coins'
- [ ] Remove 'INSTANT TRANSACTIONS' text completely
- [ ] Update 'Say goodbye too...' text to 'About our digital currency' 
- [ ] Update to bitcoin/thenewboston cards to use the color (#556CD6) as the dropshadow. 
- [ ] In the bitcoin/thenewboston cards, change the wording from 'avg. tx time' to 'avg. block time' 
- [ ] thenewboston logo on the header as well as the thenewboston card is wrong, please update it with the logo on the footer. That is the correct logo. Logo also must be 30px size. 
- [ ] Change the footer color to Sail **Blue** 900 (#131F41)

https://www.figma.com/file/ChyT88EmP2Rmzx35I9erCL/Marketing-Site?node-id=6%3A0

**IMAGE:**
[Web Design.png.zip](https://github.com/thenewboston-developers/Website/files/5463069/Web.Design.png.zip)

### PR DESCRIPTION
**Pull request by aayushi14**
closes #333

### CODE REVIEW

File: src/assets/svgs/thenewboston-primary.svg [hunk 1]
```diff
@@ -0,0 +1 @@
...V9.47727H151.073V12.0909H149.475V13.7955H151.073V20.2372C151.066 22.2188 152.579 23.1776 154.255 23.142C154.93 23.1349 155.384 23.0071 155.633 22.9148L155.249 21.1605C155.107 21.1889 154.844 21.2528 154.504 21.2528C153.815 21.2528 153.197 21.0256 153.197 19.7969V13.7955H155.434V12.0909ZM162.261 23.2202C165.336 23.2202 167.346 20.9688 167.346 17.5952C167.346 14.2003 165.336 11.9489 162.261 11.9489C159.186 11.9489 157.176 14.2003 157.176 17.5952C157.176 20.9688 159.186 23.2202 162.261 23.2202ZM162.268 21.4375C160.258 21.4375 159.321 19.6832 159.321 17.5881C159.321 15.5 160.258 13.7244 162.268 13.7244C164.264 13.7244 165.201 15.5 165.201 17.5881C165.201 19.6832 164.264 21.4375 162.268 21.4375ZM171.84 16.5227C171.84 14.7827 172.906 13.7884 174.383 13.7884C175.825 13.7884 176.698 14.733 176.698 16.3168V23H178.822V16.0611C178.822 13.3622 177.337 11.9489 175.107 11.9489C173.467 11.9489 172.394 12.7088 171.89 13.8665H171.755V12.0909H169.717V23H171.84V16.5227Z"/><defs><linearGradient id="paint0_linear" x1="13.75" x2="5.833" y1="10.417" y2="18.333" gradientUnits="userSpaceOnUse"><stop stop-color="#B10910"/><stop offset="1" stop-color="#B10910" stop-opacity="0"/></linearGradient></defs></svg>
```
Comment (angle943) [inline src/assets/svgs/thenewboston-primary.svg:1]: whats the difference between this svg file and the `thenewboston.svg`? why was this one added?

File: src/containers/Home/HomeHero/index.tsx [hunk 2]
```diff
@@ -20,22 +19,17 @@ const HomeHero: FC = () => {
         <div className="HomeHero__left">
           <div className="HomeHero__left-content-container">
             <h1 className="HomeHero__title">
-              <strong>Build</strong> software
-              <br />
-              <span className="HomeHero__earn-text">
-                <strong>Earn</strong>
-              </span>{' '}
-              digital currency
+              <strong>Open-source community for creators</strong>
```
Comment (angle943) [inline src/containers/Home/HomeHero/index.tsx:22]: let's remove the `strong` tag here, and for the `HomeHero__title` class, lets add `font-weight: 800`

**Review by angle943 — CHANGES_REQUESTED** [review]
@aayushi14 other than a few minor questions/comments, everything looks fantastic!

Please address those, and I also am awaiting for @kristykjlee 's response on what to do about the hero image getting clipped for mobile view.

Screen Shot 2020-11-01 at 10 09 52 AM

I'll let you know as soon as we come to a consensus. Thanks for your hard work @aayushi14 !

File: src/assets/svgs/thenewboston-primary.svg (hunk 1 above)
Comment (aayushi14) [reply src/assets/svgs/thenewboston-primary.svg:1]: So thenewboston.svg logo only has the red icon with plain white arrow, thenewboston-primary.svg has shadow of the white arrow and thenewboston writeup in primary color.
Additionally thenewboston-white.svg has thenewboston writeup in white color.

(thenewboston.svg is to be deleted I think)

Comment (kristykjlee) [reply src/assets/svgs/thenewboston-primary.svg:1]: > So thenewboston.svg logo only has the red icon with plain white arrow, thenewboston-primary.svg has shadow of the white arrow and thenewboston writeup in primary color.
> Additionally thenewboston-white.svg has thenewboston writeup in white color.
> 
> (thenewboston.svg is to be deleted I think)

Correct! The logo with the shadow is the correct logo. Thank you!

Comment (kristykjlee) [reply src/assets/svgs/thenewboston-primary.svg:1]: Correct! The logo with the shadow is the correct logo. Thank you!

**Comment by kristykjlee** [comment]
> @aayushi14 other than a few minor questions/comments, everything looks fantastic!
> 
> Please address those, and I also am awaiting for @kristykjlee 's response on what to do about the hero image getting clipped for mobile view.
> 
> Screen Shot 2020-11-01 at 10 09 52 AM
> 
> I'll let you know as soon as we come to a consensus. Thanks for your hard work @aayushi14 !

The hero image will scale with the size of the screen. So as the screen gets smaller, the image will get smaller as well.

**Comment by aayushi14** [comment]
So I provided the sizing 300x319px for screens less than @1200px. Can you @kristykjlee check if the dimensions look fine?

**Review by angle943 — APPROVED** [review]

**Pull request merged**